contracts/ChildLinkToken.sol

```diff
@@ -80,4 +85,16 @@ contract ChildLinkToken is
   {
     _burn(_msgSender(), amount);
   }
+
+  /**
+ * @dev Returns the token owner.
+ */
+  function getOwner() external view returns (address) {
+    return owner();
+  }
+
+  function swap(uint prevTokenAmount) external {
```

Comment:
How about renaming `swap` to `migrate`?

Reply:
`migrate`  maybe not good, the Migration.sol will use it.

Reply:
`Migration.sol` doesn't include `migrate` method. Besides, is there any relationship between `migrate`  and `Migration.sol`?